python/pyarrow/tests/test_array.py

```diff
@@ -2162,6 +2163,101 @@ def test_array_from_numpy_ascii():
     assert arrow_arr.equals(expected)
 
 
+@pytest.mark.nopandas
```

Comment:
"nopandas" is for tests that fail if Pandas is installed, I think.

Reply:
removed.  before this was testing the different return of to_pylist.